Readme.md

```diff
@@ -87,9 +84,9 @@ Format the given value in bytes into a string. If the value is negative, it is k
 
 It supports the following output formats:
 
- * `binary`: uses the binary prefixes (KiB, MiB...)
- * `decimal`|`metric`: uses the metric system (decimal) prefixes (kB, MB...)
- * `jedec`|`compatibility`: uses the binary units, but the metric prefixes (kB == 1024B, MB...)
+ * `binary`: uses the binary units (KiB, MiB...)
+ * `decimal`|`metric`: uses the metric system (decimal) units (kB, MB...)
+ * `jedec`|`compatibility`: uses the binary values, but the metric labels (kB == 1024B, MB...)
```

Comment:
"a.k.a. Windows mode" 😛

Reply:
also don't forget mbps for networking… where they use bits (which makes sense as a hardware engineer… but not form a user perspective)

Marketing loves it though!

Reply:
btw this is where the units vs prefix distinction actually made sense originally!

Since the "units" referred to the 1024/1000 distinction but the prefix referred to the Mi or M prefix

Reply:
True, now that I understand what you meant by prefixes they do fit in well in some places that I changed. My conclusion is we should clarify the meaning of "prefix" so readers get it. I'm thinking of a dissection at the top of the doc, like

![image](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fstarecat.com%2Fcontent%2Fwp-content%2Fuploads%2Fcreative-business-card-me-website-instagram-email.jpg&f=1&nofb=1)

but with unit, value, prefix, label...

Reply:
haha perfect :)

Reply:
nah, its good with your changes

Reply:
Well, I will re-read with the meaning of prefix in mind, change back where it makes more sense, and add a clarification